packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -400,8 +408,8 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
     // Only check for hovering if there is a mouse connected.
     if (_mouseIsConnected) {
       result = MouseRegion(
-        onEnter: (PointerEnterEvent event) => _showTooltip(),
-        onExit: (PointerExitEvent event) => _hideTooltip(),
+        onEnter: _handleEnter,
```

Comment:
Are these wrappers necessary, given that `MouseRegion` also wraps them? I thought that was the reason to wrap them in `MouseRegion`, so that callers don't have to.

Reply:
Ok, I reverted this change. To make style consistent, I applied the reversed change onto https://github.com/flutter/flutter/pull/37896/files#diff-f1c1016f733aebaa60581bf989e36d17L1107